nectar/src/swap.rs

```diff
@@ -716,5 +718,10 @@ async fn execute(
 
     tracing::info!("swap {} finished successfully", swap_id);
 
+    sentry::capture_message(
+        format!("Swap with swap-id {} finished successfully", swap_id).as_str(),
```

Comment:
For consistency's sake :man_shrugging: 

```suggestion
        format!("Swap {} finished successfully", swap_id).as_str(),
```